_python/test_helpers.py

```diff
@@ -14,6 +34,11 @@ def check_response(response, status_code=200, content_type=None, content_include
                 content_type = "text/html"
         assert response['content-type'].split(';')[0] == content_type
 
+    if expected_url:
+        if callable(expected_url):
```

Comment:
This workaround is probably not needed given the above?

Reply:
yep.